apps/nowcasting-app/components/utils.ts

```diff
@@ -25,18 +25,34 @@ export const getTimeFromDate = (date: Date) => {
   return date.toTimeString().split(" ")[0].slice(0, -3);
 };
 export const formatISODateString = (date: string) => {
+  // Changes the ISO date string to Europe London time, and remove timezone and seconds
+
   const dateid = date?.slice(0, 16);
```

Comment:
can you delete it if it's not necessary anymore

Reply:
Thanks, i think this is now sorted, this is now needed in 'formatISODateString'